packages/vue/src/components/profile-card.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <div class="panel">
+    <div class="title-text panel-header">
+      {{ title }}
+    </div>
+    <div class="form-container">
+      <slot />
+      <dx-form
+        ref="form"
+        :form-data="props.cardData"
+        :show-colon-after-label="true"
+        :col-count="screenInfo.isXSmall ? 2 : colCount"
+        :screen-by-width="getSizeQualifier"
+        label-location="top"
+        label-mode="outside"
+        :on-field-data-changed="evt => onFieldChange(evt.dataField, evt.value)"
+      >
+        <dxi-item
+          v-for="item in items"
+          :key="item.dataField"
+          :data-field="item.dataField"
+          :editor-type="item.editorType"
+          :editor-options="{ stylingMode: 'outlined', ...item.editorOptions } "
+          :col-span="item.colSpan"
+        >
+          <dxo-label
+            v-if="item.label"
+            :text="item.label"
+          />
+          <dx-validation-rule
+            v-for="rule in item.validators"
+            :key="rule"
+            :type="rule.type"
+          />
+          <status-select-box
+            v-if="item.dataField === 'status'"
+            :items="item.itemsList"
+            :model-value="cardData[item.dataField]"
+            label-mode="hidden"
+            @update:model-value="onFieldChange()"
+          />
+          <pictured-item-select-box
+            v-else-if="item.dataField === 'supervisor'"
+            :label="item.label"
+            :model-value="cardData[item.dataField]"
+            :items="item.itemsList"
+            @update:model-value="onFieldChange()"
+          />
+        </dxi-item>
+      </dx-form>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  DxForm,
+  DxItem as DxiItem,
```

Comment:
Why do you rename DxItem and DxLabel?

Reply:
Seems that we need to use default names of components. You took the name from angular but it is not correct for vue components.

Reply:
In docs and examples we demonstrate default component names. This may confuse users. In addition, there are no name conflicts and renaming is superfluous